po/ca.po

```diff
@@ -0,0 +1,292 @@
+
+#: src/MainWindow.vala:221
+msgid "Markdown Cheatsheet"
+msgstr "Ús del Markdown"
+
+#: src/MainWindow.vala:228 src/Widgets/Preferences.vala:29
+msgid "Preferences"
+msgstr "Preferències"
+
+#: src/MainWindow.vala:255
+msgid "Edit"
+msgstr "Edita"
+
+#: src/MainWindow.vala:256
+msgid "Preview"
+msgstr "Previsualitza"
+
+#: src/Widgets/StatusBar.vala:51
+msgid "Words: "
+msgstr "Paraules: "
+
+#: src/Widgets/StatusBar.vala:64
+msgid "Reading Time: "
+msgstr "Temps de lectura: "
+
+#: src/Widgets/StatusBar.vala:68
+msgid "Dark Mode"
+msgstr "Mode fosc"
+
+#: src/Widgets/StatusBar.vala:92
+msgid "Focus Mode"
+msgstr "Mode de concentració"
+
+#: src/Widgets/Preferences.vala:47 src/Widgets/Preferences.vala:67
+msgid "Editor"
+msgstr "Editor"
+
+#: src/Widgets/Preferences.vala:48 src/Widgets/Preferences.vala:177
+msgid "Interface"
+msgstr "Interfície"
+
+#. Close button
+#: src/Widgets/Preferences.vala:50 src/Widgets/Cheatsheet.vala:51
+msgid "Close"
+msgstr "Tanca"
+
+#: src/Widgets/Preferences.vala:69
+msgid "Show Line Numbers:"
+msgstr "Mostra els números de línia"
+
+#: src/Widgets/Preferences.vala:71
+msgid "Enable Spellchecking:"
+msgstr "Habilita el corrector ortogràfic:"
+
+#: src/Widgets/Preferences.vala:75
+msgid "Geometry"
+msgstr "Geometria"
+
+#: src/Widgets/Preferences.vala:76
+msgid "Spacing of Text:"
+msgstr "Espaiat del text:"
+
+#: src/Widgets/Preferences.vala:78 src/Widgets/Preferences.vala:118
+msgid "Small"
+msgstr "Petit"
+
+#: src/Widgets/Preferences.vala:79 src/Widgets/Preferences.vala:119
+msgid "Normal"
+msgstr "Normal"
+
+#: src/Widgets/Preferences.vala:80 src/Widgets/Preferences.vala:120
+msgid "Large"
+msgstr "Gran"
+
+#: src/Widgets/Preferences.vala:116
+msgid "Margins of Text:"
+msgstr "Marges del text:"
+
+#: src/Widgets/Preferences.vala:178
+msgid "Show the Save Button:"
+msgstr "Mostra el botó Desa:"
+
+#: src/Widgets/Preferences.vala:181
+msgid "Mode"
+msgstr "Mode"
+
+#: src/Widgets/Preferences.vala:182
+msgid "Enable Focus Mode:"
+msgstr "Habilita el mode de concentració:"
+
+#: src/Widgets/Preferences.vala:184
+msgid "Enable Dark Mode:"
+msgstr "Habilita el mode fosc:"
+
+#: src/Widgets/Preferences.vala:187
+msgid "Font"
+msgstr "Tipus de lletra"
+
+#: src/Widgets/Preferences.vala:188
+msgid "Custom font:"
+msgstr "Tipus de lletra personalitzat:"
+
+#: src/Widgets/Preferences.vala:198
+msgid "Statusbar"
+msgstr "Barra d’estat"
+
+#: src/Widgets/Preferences.vala:199
+msgid "Show Statusbar:"
+msgstr "Mostra la barra d’estat:"
+
+#: src/Widgets/Cheatsheet.vala:28
+msgid "Cheatsheet"
+msgstr "Ús"
+
+#: src/Widgets/Cheatsheet.vala:46
+msgid "Text"
+msgstr "Text"
+
+#: src/Widgets/Cheatsheet.vala:47
+msgid "Links & Special"
+msgstr "Enllaços i text especial"
+
+#: src/Widgets/Cheatsheet.vala:48 src/Widgets/Cheatsheet.vala:137
+msgid "Tables"
+msgstr "Taules"
+
+#: src/Widgets/Cheatsheet.vala:79
+msgid "Header"
+msgstr "Encapçalament"
+
+#: src/Widgets/Cheatsheet.vala:80
+msgid "# Header 1"
+msgstr "# Encapçalament 1"
```

Comment:
I’ve translated these strings as if the original ones said “headings”; *header* in Catalan is *capçalera*. The same goes for the Spanish translation below.

Reply:
If it doesn't make people confused, this is ok.